src/scss/sections/_product-waitlist.scss

```diff
@@ -453,3 +453,32 @@
 		right: 8px;
 	}
 }
+
+.product-waitlist-form {
+	&-oos {
+		border-radius: 20px;
+
+		@include media-breakpoint-down(md) {
+			.input-group .form-control,
+			.input-group .btn {
+				border-radius: $border-radius;
```

Comment:
for border-radius use the default utility class directly on the DOM

Reply:
border radius for desktop and mobile is different, so cannot directly using the utility class